src/unstract/adapters/vectordb/postgres/src/postgres.py

```diff
@@ -58,6 +59,9 @@ def get_vector_db_instance(self) -> BasePydanticVectorStore:
 
     def _get_vector_db_instance(self) -> BasePydanticVectorStore:
         try:
+            encrypted_password = urllib.parse.quote_plus(
```

Comment:
Do you mean encoded password? @harini-venkataraman

Reply:
Also, are we having this kind of construct anywhere where we connect to our postgres database (where we store unstract data)?

Reply:
Guess the same code construct is also required for Supabase. Supabase is not in use. However for sanity sake, we can make the changes there also.

Reply:
@gaya3-zipstack Yes, such connection string construct is used in Supabase. Updated there as well.